Dockerfile

```diff
@@ -15,6 +15,17 @@ RUN add-apt-repository \
 RUN apt-get update
 RUN apt-get install -y docker-ce=17.03.0~ce-0~debian-jessie
 
+# Install libsodium
+RUN apt-get install -y build-essential
+
+RUN curl https://download.libsodium.org/libsodium/releases/libsodium-1.0.12.tar.gz \
+    | tar xz 
+
+RUN cd libsodium-1.0.12/ \
+	&& ./configure \
+	&& make && make check \
+	&& make install
+
```

Comment:
I know I said the opposite yesterday, but actually it does seem like it would be good to run `./scripts/install-libsodium.sh` here. I had forgotten that we would still need that script for developers installing on their own machine. At which point, if we can use the same script in both places that's good, as it means the script developers will use is being tested on TeamCity.

So the docker file becomes something like:
```
COPY ./scripts/install-libsodium.sh .
RUN ./install-libsodium.sh
```